src/libs/onScreenTransitionEnd/index.js

```diff
@@ -0,0 +1,14 @@
+import Str from 'expensify-common/lib/str';
+
+/**
+ * Call the callback after screen transiton has ended
+ *
+ * @param {Object} navigation Screen navigation prop
+ * @param {Function} callback Method to call
+ * @returns {Function}
+ */
+function onScreenTransitionEnd(navigation, callback) {
+    return navigation.addListener('transitionEnd', evt => Str.result(callback, evt));
```

Comment:
Why `Str.result()` vs `navigation.addListener('transitionEnd', callback)` ?

Reply:
Just to skip the test if callback is present. But yeah we can jus pass the callback directly.

Reply:
I will add these in a new PR.

Reply:
dropped a PR #3228. Thanks.